shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/SurveyQuestionViewModelTest.kt

```diff
@@ -17,32 +27,81 @@ import kotlin.test.Test
 @ExperimentalCoroutinesApi
 class SurveyQuestionViewModelTest {
 
-    private val mainThreadSurrogate = newSingleThreadContext("UI thread")
+    @Mock
+    private val useCase = mock(classOf<SubmitSurveyAnswerUseCase>())
+
     private lateinit var viewModel: SurveyQuestionViewModel
 
     @BeforeTest
     fun setUp() {
-        Dispatchers.setMain(mainThreadSurrogate)
+        Dispatchers.setMain(TestDispatchersProvider.io)
```

Comment:
Same here, I think we don't need it anymore.
```suggestion
```

Reply:
This was solved by rebasing